roles/rsshub/templates/docker-compose.rsshub.yml.j2

```
@@ -13,9 +13,7 @@ services:
     networks:
       - traefik_network
     environment:
-      # OPTIONAL: http basic auth
-      # - HTTP_BASIC_AUTH_NAME={{ default_username }}
-      # - HTTP_BASIC_AUTH_PASS="{{ default_password }}"
+      - TZ=Etc/UTC
```

We got a variable for this 

```suggestion
      - TZ={{ common_timezone }}
```